cli/src/test/scala/co/uproot/abandon/CliTest.scala

```diff
@@ -42,8 +42,8 @@ class CliAppErrors extends DefaultArgsDirSuite {
   /**
     * these should fail, but they are not failing
     */
-  ignoreDirSuiteTestCases(errorRoot, Glob("bugsInputError/**.exec")) { args: Array[String] =>
-    assertThrows[SettingsError] {
+  runDirSuiteTestCases(errorRoot, Glob("bugsInputError/**.exec")) { args: Array[String] =>
+    assertThrows[InputPosError] {
```

Comment:
As part of the fix, this testcase should move to a different directory: `inputError`. I think the `bugsInputError` directory is for bugs in the error reporting code (which we are fixing now).

Apart from moving to the `inputError` directory, no other changes should be required.